src/styles/utils/_extends.scss

```
@@ -0,0 +1,12 @@
+%page-flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+%centered-content {
+  position: fixed;
```

is it ok if you change to absolute?
```suggestion
  position: absolute;
```